backend/app/api/services/thread_service.py

```diff
@@ -0,0 +1,87 @@
+import uuid
+from typing import Optional
+
+from app.api.schemas import Filter, Message, MessageCreate, SupportCreate, SupportThread
+
+_threads: list[SupportThread] = []
+
+
+class SupportThreadService:
+    """
+    class for manipulation SupportThread entities.
```

Comment:
In that case, it probably should be a `SupportThreadManager`/`SupportThreadRepository`, rather than `SupportThreadService`. Is it doing something other than intelligent storage?

Reply:
Agree with you, I will change service part in names with repositiry